antora-playbook.yml

```diff
@@ -13,6 +13,8 @@ runtime:
     format: pretty
     # set to info to get details from the Antora extensions
     level: info
+    # Antora exits with a non-zero exit code if an error is logged -> https://docs.antora.org/antora/latest/playbook/runtime-log-failure-level
+    failure_level: error
```

Comment:
⚠️ This will also make the production build fail if some errors are introduced. Currently, we have no notification for such errors. We may need to introduce slack notification to ensure we are notified (in another PR).